src/components/mobile/ExploreMobile.tsx

```diff
@@ -13,8 +13,12 @@ import Legend from "components/map/Legend";
 import Translate from "utils/translate/translateService";
 import WhoLogo from "components/shared/WhoLogo"
 
+interface props {
+  pulsateUnityFilter: boolean;
+  setPulsateUnityFilter: React.Dispatch<React.SetStateAction<boolean>>;
+}
 
-export default function ExploreMobile() {
+export default function ExploreMobile({ pulsateUnityFilter, setPulsateUnityFilter } : props) {
```

Comment:
Why are we doing all of this prop drilling with `pulsateUnityFilter`? Couldn't we just make this a property of the context and interact with it via the context API?